### ISSUE TITLE
Windows binaries are not packaged with the application correctly

### ISSUE THREAD

**Issue by nwalsh1995** [mobius-software-ltd/pyton3-dtls#2]
In the original dtls it packages its own .dll files. From the setup.py there are the instructions:
```python
    # Full upload sequence for new version:
    #    1. python setup.py bdist_wheel
    #    2. python setup.py bdist_wheel -p win32
    #    3. python setup.py bdist_wheel -p win_amd64
    #    4. twine upload dist/*
```
If you try to run the windows commands they break due to a few errors from `setup.py` python 3 incompatibilities.

**Comment by hugonun** [comment]
Can confirm. python3-dtls does NOT work on Windows because of this.

**Comment by yulianoifa-mobius** [comment]
Fixed by commit 7663c663b8c7faae92aed23ebfc73f67a91bbded
